tachyon/zk/plonk/circuit/rotation.h

```diff
@@ -50,15 +50,6 @@ class TACHYON_EXPORT Rotation {
     return size_t{base::bit_cast<uint32_t>(value % size)};
   }
 
-  template <typename Domain, typename F = typename Domain::F>
-  F RotateOmega(const Domain* domain, const F& point) const {
-    if (value_ >= 0) {
-      return point * domain->group_gen().Pow(value_);
-    } else {
-      return point * domain->group_gen_inv().Pow(-1 * value_);
-    }
-  }
```

Comment:
```suggestion
  template <typename Domain, typename F = typename Domain::F>
  F RotateOmega(const Domain* domain, const F& point) const {
    return point * domain->GetElement(value_);
  }
```
I think it's better to keep `RotateOmega` as `Rotation`. How about just fixing the code? (GetElement already handles with the signs.)